packages/react-router/test/base/src/pages/relative-paths/RelativePaths.tsx

```diff
@@ -0,0 +1,131 @@
+import {
+  IonContent,
+  IonHeader,
+  IonPage,
+  IonTitle,
+  IonToolbar,
+  IonRouterOutlet,
+  IonList,
+  IonItem,
+  IonLabel,
+  IonBackButton,
+  IonButtons,
+} from '@ionic/react';
+import React from 'react';
+import { Route } from 'react-router-dom';
+
+/**
+ * This test page verifies that IonRouterOutlet correctly handles
+ * relative paths (paths without a leading slash) the same way
+ * React Router 6's Routes component does.
+ *
+ * Issue: https://github.com/ionic-team/ionic-framework/issues/24177#issuecomment-3624311206
```

Comment:
```suggestion
```

I understand linking issues for regressions but I don't think we need to link to issue comments for basic functionality that existed in rr5.